modular_skyrat/modules/morenarcotics/code/thc.dm

```diff
@@ -1,8 +1,12 @@
-/obj/item/food/grown/cannabis/on_grind()
-	. = ..()
-	if(HAS_TRAIT(src, TRAIT_DRIED))
-		grind_results = list(/datum/reagent/drug/thc/hash = 0.15*src.seed.potency)
-		reagents.clear_reagents() //prevents anything else from coming out
```

Comment:
The reason this borks is because the math relies on there being stuff (specifically, nutriment) inside the plant to make stuff. Which skyrat, in their infinite wisdom, removes with the `reagents.clear_reagents()`.

Update: Actually turns out this code isn't even needed - normal cannabis already contains only thc when dried so this just.. tries to do the exact same thing that would've already happened.

Reply:
What the fuck. Well, I guess you could try and rewrite this into
```
reagents.clear_reagents()
reagents.add_reagent(/datum/reagent/drug/thc/hash, 0.15 * seed.potency)
```
then, should work about fine hopefully

Reply:
I can't believe I didn't think of that. I guess I got so in my own head about the stupid way they did it. Will amend when home.